typings/glamorous.d.ts

```diff
@@ -3,137 +3,33 @@
 // Definitions by: Kok Sam <https://github.com/sammkj>
 
 import * as React from "react";
-import { StatelessComponent, ComponentClass, PureComponent } from "react";
+import { ComponentClass, CSSProperties, StatelessComponent, SVGAttributes } from 'react'
```

Comment:
Actually, we should follow the standard of the project. Could you use single quotes only please?